README.md

```diff
@@ -293,21 +293,21 @@ which will give us WebSocket functionality.
 Then add the following JavaScript ("Client") code:
 
 ```js
-var channel = socket.channel('room:lobby', {}); // connect to chat "room"
+let channel = socket.channel('room:lobby', {}); // connect to chat "room"
```

Comment:
@kyleboe our only reasoning for avoiding using `let` was to support browsers that don't support it.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let#Browser_compatibility
Sadly, Internet explorer 10 is still used according to recent stats ...
but given that this tutorial/example is targeted at "modern browsers" happy to go with `let`. 👍